clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -8722,7 +8722,7 @@ void SPIRVTranslator::ConstructJob(Compilation &C, const JobAction &JA,
   TranslatorArgs.push_back("-o");
   TranslatorArgs.push_back(Output.getFilename());
   if (JA.isDeviceOffloading(Action::OFK_SYCL)) {
-    TranslatorArgs.push_back("-spirv-max-version=1.3");
+    TranslatorArgs.push_back("-spirv-max-version=1.4");
```

Comment:
@AGindinson, why do we set this option twice?

Reply:
One for translation of the device code (clang.cpp) another one is for possible linking on SPIR-V level (sycl.cpp, but actually there it's a dead code now). At some point I resolve a [comment](https://github.com/intel/llvm/pull/4357#discussion_r695233786) from another PR and we'll set SPIR-V information in a one place.